### DISCUSSION TITLE
The "images.domains" configuration is deprecated. Please use "images.remotePatterns" configuration instead.

### DISCUSSION THREAD

**Discussion by shabbir-ahmed** [vercel/next.js#57747] [App Router] [answered]
I recently upgraded one of my Next.js projects from `v13.4.13` to `v14.0.0`, and I encountered the following error in the console:

>   ⚠ The "images.domains" configuration is deprecated. Please use the "images.remotePatterns" configuration instead.

Could you please explain what this error message means? I tried searching the web for information but couldn't find anything helpful.

I tried with 

     images: {
            remotePatterns: ["southeast-2.amazonaws.com"],
     }

Instead of this

     images: {
            domains: ["southeast-2.amazonaws.com"],
     }

But it did not get solved.

Thank you!

**Comment by icyJoseph** [comment, marked as answer]
It wants you to do this:

- https://nextjs.org/docs/app/building-your-application/optimizing/images#remote-images

```js
module.exports = {
  images: {
    remotePatterns: [
      {
        protocol: 'https',
        hostname: 's3.amazonaws.com',
        port: '',
        pathname: '/my-bucket/**',
      },
    ],
  },
}
```

In your case with southeast-2, etc, etc

**Reply by shabbir-ahmed** [reply]
It's my mistake! I should have searched in the documentation. 

Thank you!
